mongodb/src/main/kotlin/com/example/plugins/Routing.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.plugins
+
+import com.example.entities.*
+import com.example.service.ArticleService
+import io.ktor.http.*
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+
+fun Application.configureRouting() {
+    val articleService = ArticleService()
```

Comment:
I would also create a service in `main` and pass it here as an argument.

Reply:
Please close the service when the server goes down. You can subscribe on ApplicationStopped event: https://ktor.io/docs/events.html#handle-events-application